lib/Table/createLongitudeLatitudeFeaturePerId.ts

```diff
@@ -285,42 +385,49 @@ function createFeature(
 
   const show = calculateShow(availability);
   const feature = new TerriaFeature({
-    position: positionProperty,
+    position:
+      positionProperty instanceof PreSampledPositionProperty
+        ? positionProperty.getProperty()
+        : positionProperty,
```

Comment:
Is this good for code maintainability moving forward? Maybe just a comment. 'If this is not an instance of PreSampled... then it must be a SampledPositionProperty' or something

Reply:
done!